deviceComunicationDocumentation.MD

```diff
@@ -429,57 +404,56 @@ Při detekci standartní chyby dioda vždy blikne 3x.
-| 00000000 00000000 00000010 00000000 | 2 - Vložení baterie která není plně nabitá. Kontroluje se pouze po hard restartu | 4x                   |
+| 00000000 00000000 00000010 00000000 | 2 - Vložení baterie, která není plně nabitá. Kontroluje se pouze po hard restartu | 4x                   |
 | 00000000 00000000 00000100 00000000 | 3 - Stavový automat se dostal do neznámého stavu                                 | 3x                   | 
 | 00000000 00000000 00001000 00000000 | 4 - Zařízení se nedokáže pripojit k siti po standartním restartu                 | 6x                   |
 | 00000000 00000000 00010000 00000000 | 5 - Standartní chyba nastala 5*                                                  | 3x                   |
 
-Pokud chyba s číslem 1,3 a 5 nastane 4 hodiny po restartu tak zařízení postupuje následujícím způsobem:
+Pokud chyba s číslem 1,3 a 5 nastane 4 hodiny po restartu, tak zařízení postupuje následujícím způsobem:
 
 1. Dioda v deseti cyklech Xkrát blikne (podle tabulky) pro oznámení erroru.
-2. Zařízení se restartuje a dále funguje běžným způsobem jako by bylo restartováno (TODO odkaz)
+2. Zařízení se restartuje a dále funguje běžným způsobem jako by bylo [restartováno](#standartní-restart)
 3. Pokud error stále přetrvává tak se znovu opakuje zpracování erroru
 
 Pokud chyba s číslem 1,3,4 a 5 nastane do 4 hodin od restartu tak zařízení postupuje následujícím způsobem:
 
 1. Dioda v deseti cyklech Xkrát blikne (podle tabulky) pro oznámení erroru.
-2. Zařízení zkontroluje jestli uběhly 4 hodiny od restartu
-3. Pokud neuběhly tak se uspí na 2 minuty a následně opakuje bod 1. a 2,
-4. Pokud uběhly tak se zařízení restartuje a dále funguje běžným způsobem jako by bylo restartováno (TODO odkaz)
-5. Pokud error stále přetrvává tak se znovu opakuje zpracování erroru
+2. Zařízení zkontroluje jestli uběhly 4 hodiny od restartu.
+3. Pokud neuběhly tak se uspí na 2 minuty a následně opakuje bod 1. a 2.
+4. Pokud uběhly tak se zařízení restartuje a dále funguje běžným způsobem jako by bylo [restartováno](#standartní-restart).
+5. Pokud error stále přetrvává tak se znovu opakuje zpracování erroru.
 
 
 Pro chybu s číslem 2 se zařízení chová podle následujícího seznamu:
 
-1. Uživatel vyvolá hard restart (TODO odkaz)
-2. Dioda 1x blikne aby oznámila restart
-3. Dioda 1x blikne aby oznámila inicializaci
-4. Zařízení v inicializaci zjistí že baterie neni plně nabitá
-5. Dioda v deseti cyklech 4krát blikne (podle tabulky) pro oznámení erroru
-6. Zařízení v dvouminutových intervalech provádí bod 5. následující 4 hodiny
-7. Zařízení se restartuje
-8. Dioda 1x blikne aby oznámila restart
-9. Dioda 1x blikne aby oznámila inicializaci
-10. Inicializace se dokončí. Detekce erroru 2 (Vložení baterie která není plně nabitá) už neprobíhá.
-11. Zařízení odešle restartovací zprávu
-12. Zařízení odešle errorovou zprávu
-13. Zařízení odešle testovací zprávu
-14. Zařízení funguje běžným způsobem
+1. Uživatel vyvolá [hard restart](#hard-restart).
```

Comment:
To je divné, ten postup nesouhlasí s tím že při hard restartu se zařízení přepne do transportu, takže nemůže nakonci odeslat error, test a začít fungovat běžným způsobem ...